sqlframe/base/dataframe.py

```diff
@@ -1935,6 +1935,11 @@ def count(self) -> int:
     def createGlobalTempView(self, name: str) -> None:
         raise NotImplementedError("Global temp views are not yet supported")
 
+    def isEmpty(self) -> bool:
+        from sqlframe.base import functions as F
+
+        return True if self.select(F.lit(True)).head(1) == [] else False
```

Comment:
Once the other PR is updated, this can be changed to `return not bool(self.select(F.lit(True)).head())`. If a row is returned, then `bool()` will return True and therefore the `not` will make it False which is correct since it is not empty.